prov/efa/test/efa_unit_test_mocks.c

```diff
@@ -145,8 +145,21 @@ void efa_mock_efa_qp_wr_send_verify_handshake_pkt_local_host_id_and_save_wr(stru
 	struct efa_rdm_pke* pke;
 	struct efa_rdm_base_hdr *efa_rdm_base_hdr;
 	uint64_t *host_id_ptr;
+	uint64_t wr_id = qp->ibv_qp_ex->wr_id;
+
+	/* Duplicate the logic from efa_rdm_cq_get_pke_from_wr_id and assume the
+	 * completion is not from an unsolicited write recv. This function is
+	 * mocking efa_qp_wr_send, so it cannot accept the CQ or the resource as
+	 * an input parameter */
+#if ENABLE_DEBUG
```

Comment:
nit: u can reuse the static inline function you created

Reply:
No, I can't reuse it inside the mock. Because the mock cannot have the ibv_cq as an input argument. I explained in the comment.